FinniversKit/Sources/Components/Ribbon/Models/RibbonViewModel.swift

```diff
@@ -4,12 +4,29 @@
 
 import UIKit
 
-public struct RibbonViewModel: Hashable {
+public struct RibbonViewModel: Hashable, Codable {
     public let style: RibbonView.Style
     public let title: String
-
+    
     public init(style: RibbonView.Style, title: String) {
         self.style = style
         self.title = title
     }
+    
+    private enum CodingKeys: String, CodingKey {
+        case style
+        case title
+    }
+    
+    public init(from decoder: Decoder) throws {
```

Comment:
Do we actually need custom decoder here? Seems like default provided by using Codable implementation is enough here

Reply:
indeed, removed